README.md

```diff
@@ -48,11 +74,15 @@ Sketches provided:
 - paj7620_9_gestures    // Polls for all 9 gestures
 - paj7620_interrupt     // Sets up an interrupt routine to wait for any gestures
 - paj7620_wave_count    // Polls for the number of waves the sensor sees
+- paj7620_cursor_demo   // Polls the sensor for a 'cursor', which is an object in front of the sensor. It then pulls the cursor location using (x,y) coordinates
```

Comment:
You can document this example with Doxygen in the `examples/examples.doc` file. All that's needed is to add the necessary documentation comment (`/** \example paj7620_cursor_demo.ino...*/`) to that file. Unfortunately, Doxygen doesn't seem to parse any comments from the examples themselves, so this is a bit of a workaround to avoid having them in the source code.